### PR TITLE
Adding link to document for NMI in Azure

### PR DESCRIPTION
**Pull request by joinimran**
Add a link for the user to point to how to use NMI in the Azure environment. 

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/6223

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/client-management/generate-kernel-or-complete-crash-dump.md [hunk 1]
```diff
@@ -101,6 +101,8 @@ To do this, follow these steps:
 
 9. Test this method on the server by using the NMI switch to generate a dump file. You will see a STOP 0x00000080 hardware malfunction.
 
+If you want to run NMI in Microsoft Azure using serial console, see [Use Serial Console for SysRq and NMI calls](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/serial-console-nmi-sysrq).
```
Comment (JohanFreelancer9) [inline windows/client-management/generate-kernel-or-complete-crash-dump.md]:
```suggestion
If you want to run NMI in Microsoft Azure using Serial Console, see [Use Serial Console for SysRq and NMI calls](https://docs.microsoft.com/azure/virtual-machines/linux/serial-console-nmi-sysrq).
```
I would also suggest changing this to a NOTE.

Comment (joinimran) [reply windows/client-management/generate-kernel-or-complete-crash-dump.md]: Will that not create a messy look in the text? Keeping this in view I avoided note.

**Comment by joinimran** [comment]
@e0i Ready to go live.

**Pull request merged**